wallets/provider-mytonwallet/src/helpers.ts

```diff
@@ -1,3 +1,9 @@
+import { Address } from '@ton/core';
+
 export function myTonWallet() {
   return window.mytonwallet?.tonconnect ?? null;
 }
+
+export function parseAddress(rawAddress: string): string {
+  return Address.parse(rawAddress).toString({ bounceable: false });
```

Comment:
I guess it is a backend requirement but can explain why it should be non-bouncable? for the record.

Reply:
No, that's not exactly the reason. I checked with the backend team, and they confirmed it doesn't make a difference for them. However, based on the documentation, I updated the address to a non-bounceable format. Additionally, if you connect using the `MyTonWallet` provider, you'll notice that the address shown in the provider extension now matches the one displayed in our app.
https://docs.ton.org/v3/documentation/smart-contracts/addresses#bounceable-vs-non-bounceable-addresses